Formula/devil.rb

```diff
@@ -49,6 +49,9 @@ class Devil < Formula
   depends_on "libtiff"
   depends_on "little-cms2"
 
+  # allow compiling against jasper >= 3.0.0
+  patch :DATA
```

Comment:
Noting that this patch may not be sufficient. I only added quick fixes for compilation, but I see that JasPer has mentioned in [NEWS](https://jasper-software.github.io/jasper-manual/latest/html/index.html#overview__news) that `jas_init()` no longer registers `jas_cleanup()`, so this may lead to memory leaks and incorrect garbage collection.

Reply:
I'd keep the patch and make a PR. Then upstream van decide what to do with the rest.

Reply:
Sure. Can upstream compilation fix and add a note in either the PR or separate issue that API changes for JasPer 3.0 may need some additional testing for runtime behavior (memory leaks/etc).

Probably can come back to this if Homebrew user points out any issues in DevIL.